### DISCUSSION TITLE
How to close workspace from Launcher?

### DISCUSSION THREAD

**Discussion by lucasrabiec** [wezterm/wezterm#3680] [Q&A] [answered]
Is there any way to close workspaces from the workspaces list (`action.ShowLauncherArgs({ flags = "WORKSPACES" })`) or any other fast way (without entering it and closing each pane in it)?

If not are there plans to make such a feature?

**Comment by wez** [comment, marked as answer]
Not currently.
See:
* https://github.com/wez/wezterm/issues/3658
